README.md

```diff
@@ -14,12 +14,22 @@ Available variables are listed below, along with default values (see `defaults/m
 
     fluentd_package_state: present
 
+Extra groups to add log access, secret:
+    fluentd_groups:
+    - adm
+
 The `td-agent` Fluentd package state; set to `latest` to upgrade or change versions.
 
+    fluentd_adm_group: false
```

Comment:
I am not seeing where you are using it :eyes:

Reply:
Not anymore, incorporated in `fluentd_extra_groups`